package.json

```diff
@@ -19,6 +19,7 @@
     "prepare": "yarn contracts:compile && yarn graphql:fetch && yarn graphql:generate && yarn i18n:compile",
     "start": "craco start",
     "build": "craco build",
+    "build:instrumented": "REACT_APP_UNSAFE_EVAL=true craco --max_old_space_size=4096 build",
```

Comment:
i'd prefer to name this something like `REACT_APP_OVERRIDE_CSP` or something, potentially with the value being the new CSP rule vs UNSAFE_EVAL. this current name requires knowledge of the CSP rule unsafe-eval, so i'd like it to be more obvious what the env variable is doing (overriding the csp).